docs/learning/ui/AugmentingUI.md

```diff
@@ -34,41 +66,9 @@ export class MyUiItemProvider {
 Register the UiItemsProvider.
 
 ```ts
-UiItemsManager.register(new MyUiItemProvider(IModelApp.i18n));
+UiItemsManager.register(new MyUiItemProvider(iTwinApp.i18n));
```

Comment:
This instance of IModelApp shouldn't be changed.

Reply:
Oops! I thought I bypassed all the IModelApp instances.

Reply:
I take it  back: this is what I get when I do an npm start without installing:

Failed to compile.

./node_modules/@microsoft/applicationinsights-web/dist-esm/applicationinsights-web.js
Attempted import error: 'PerfEvent' is not exported from '@microsoft/applicationinsights-core-js'.

Reply:
Oops! I thought I skipped all the IModelApp instances in code.